Examples/GenerativeAISample/ChatSample/Views/ErrorDetailsView.swift

```diff
@@ -157,6 +157,22 @@ struct ErrorDetailsView: View {
             )
           }
 
+        case GenerateContentError.unsupportedUserLocation:
+          Section("Error Type") {
+            Text("Unsupported User Location")
+          }
+
+          Section("Details") {
+            SubtitleFormRow(title: "Error description", value: error.localizedDescription)
+            SubtitleMarkdownFormRow(
+              title: "Help",
+              value: """
+              The API is unsupported in your location (country / territory); please see
+              [available regions](https://ai.google.dev/available_regions#available_regions).
```

Comment:
... see the list of available regions ...

Reply:
SGTM, updated. Thanks!